apps/ledger-live-desktop/tests/enum/Account.ts

```diff
@@ -344,7 +344,7 @@ export class Account {
     Currency.TRX_USDT,
     "Tron 1",
     "TDUKFB9wj3P5f2iNvkRuaDDeWVkTdUVhs1",
-    AccountType.TRC20,
+    undefined,
```

Comment:
[ASK] Why do we remove this ?

Reply:
discussed with @VicAlbr, account isn't a `child` account (at least atm this value isn't in use, I created `AccountType` enum mostly for `erc20 edge cases on swaps`)
I can add it back iyw, but it's not in use atm